package.json

```diff
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "kolorist": "^1.8.0",
+    "latest-version": "^5.1.0",
```

Comment:
This pulls in a total of 19 dependencies to do a simple fetch call, see https://npm.anvaka.com/#/view/2d/latest-version . That's a bit much to fetch some json from the npm api.

Reply:
yep i agree, added this just to get it to work but will look into replacing this with something better

Reply:
i've changed it so it pulls the latest version from the `meta.json` given by the jsr using fetch. let me know if you think it's a good idea or not.